downloader.py

```diff
@@ -10,6 +10,24 @@
 __log__ = logging.getLogger(__name__)
 
 
+def download_media(client, msg, target_id):
+    media = msg.media
```

Comment:
I would add a few more checks here, like `if isinstance(msg, Message)` then you get the message media otherwise you already have it.